ci-operator/jobs/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master-presubmits.yaml

```diff
@@ -128,6 +128,70 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-serial,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```

Comment:
for now, not sure how stable the new job will be. In the future, we are going to always test against `SNO`